octavia_f5/controller/worker/tasks/f5_tasks.py

```diff
@@ -272,10 +278,75 @@ def execute(self, bigip: bigip_restclient.BigIPRestClient,
         return device_route
 
 
+class SyncSubnetRoutes(task.Task):
+    """ Task to create static subnet routes """
+
+    @decorators.RaisesIControlRestError()
+    def execute(self, bigip: bigip_restclient.BigIPRestClient,
+                selfips: [network_models.Port],
+                network: f5_network_models.Network):
+
+        # Skip passive device if route_on_active is enabled
+        if CONF.networking.route_on_active and not bigip.is_active:
+            return None
+
+        def subnet_in_selfips(subnet, selfips):
+            for selfip in selfips:
+                for fixed_ip in selfip.fixed_ips:
+                    if fixed_ip.subnet_id == subnet:
+                        return True
+            return False
+
+        # Fetch existing routes
+        response = bigip.get(path=f"/mgmt/tm/net/route?$filter=partition+eq+Common").json()
+        existing_routes = response.get('items', [])
+
+        # subnet routes that must exist (routes already exist for SelfIPs)
+        subnets_that_need_routes = [subnet for subnet in network.subnets if not subnet_in_selfips(subnet, selfips)]
+
+        # delete existing subnet routes that aren't needed anymore - we'll only provision the missing ones
+        for existing_route in existing_routes:
+            existing_route_name = existing_route['name']
+
+            # ignore routes that are not subnet routes of this network
+            subnet_route_network_part = get_subnet_route_name(network.id, '')
+            if not existing_route_name.startswith(subnet_route_network_part):
+                continue
+
+            # delete the route or ignore it if it's still needed
+            delete_device_route = True
+            for subnet_that_needs_route in subnets_that_need_routes:
+
+                # if the route is a needed subnet route, don't touch it
+                if existing_route_name == get_subnet_route_name(network.id, subnet_that_needs_route):
+                    subnets_that_need_routes.remove(subnet_that_needs_route)
+                    delete_device_route = False
+
+            # Delete unneeded route
+            if delete_device_route:
+                res = bigip.delete(path=existing_route['path'])
```

Comment:
there is no path element in the bigip response:
```
2023-03-03 12:17:50 21842 ERROR octavia_f5.controller.worker.l2_sync_manager Traceback (most recent call last):
2023-03-03 12:17:50 21842 ERROR octavia_f5.controller.worker.l2_sync_manager   File "/Users/d072895/.virtualenvs/yoga_python3.8.10/lib/python3.8/site-packages/taskflow/engines/action_engine/executor.py", line 53, in _execute_task
2023-03-03 12:17:50 21842 ERROR octavia_f5.controller.worker.l2_sync_manager     result = task.execute(**arguments)
2023-03-03 12:17:50 21842 ERROR octavia_f5.controller.worker.l2_sync_manager   File "/Users/d072895/.pyenv/versions/3.8.10/lib/python3.8/contextlib.py", line 75, in inner
2023-03-03 12:17:50 21842 ERROR octavia_f5.controller.worker.l2_sync_manager     return func(*args, **kwds)
2023-03-03 12:17:50 21842 ERROR octavia_f5.controller.worker.l2_sync_manager   File "/Users/d072895/Workspace/yoga/octavia-f5-provider-driver/octavia_f5/controller/worker/tasks/f5_tasks.py", line 327, in execute
2023-03-03 12:17:50 21842 ERROR octavia_f5.controller.worker.l2_sync_manager     res = bigip.delete(path=existing_route['path'])
2023-03-03 12:17:50 21842 ERROR octavia_f5.controller.worker.l2_sync_manager KeyError: 'path'
```

I assume you meant fullPath?